include/nbl/builtin/glsl/limits/numeric.glsl

```diff
@@ -39,22 +39,7 @@
 #define	nbl_glsl_FLT_EPSILON 5.96046447754e-08
 #endif 
 
-
-// TODO: move to some other header (maybe ieee754.glsl)
-uint nbl_glsl_ieee754_extract_biased_exponent(float x)
-{
-	return bitfieldExtract(floatBitsToUint(x),23,8);
-}
-float nbl_glsl_ieee754_replace_biased_exponent(float x, uint exp_plus_bias)
-{
-	return uintBitsToFloat(bitfieldInsert(floatBitsToUint(x),exp_plus_bias,23,8));
-}
-// performs no overflow tests, returns x*exp2(n)
-float nbl_glsl_ieee754_fast_mul_exp2(float x, int n)
-{
-	return nbl_glsl_ieee754_replace_biased_exponent(x,nbl_glsl_ieee754_extract_biased_exponent(x)+uint(n));
-}
-
+#include <nbl/builtin/glsl/ieee754.glsl>
```

Comment:
not sure you need that include here....

Reply:
So forward declare `nbl_glsl_ieee754_fast_mul_exp2`?

Reply:
ah ok, now we're sure you need it ;)